src/main/java/com/example/bookshop/dto/BookDto.java

```diff
@@ -0,0 +1,16 @@
+package com.example.bookshop.dto;
+
+import java.math.BigDecimal;
+import lombok.Data;
+
```

Comment:
maybe will be better to use separate annotation as @Getter @Setter @ToString ect